expath-pkg.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-<package xmlns="http://expath.org/ns/pkg" name="http://existsolutions.com/apps/tei-publisher" abbrev="tei-publisher" version="7.0.0" spec="1.0">
+<package xmlns="http://expath.org/ns/pkg" name="http://existsolutions.com/apps/tei-publisher" abbrev="tei-publisher" version="8.0.0-SNAPSHOT" spec="1.0">
```

Comment:
I would not update the version in a PR

Reply:
Since there is no bot here doing if for us I did it manually. With the divergent history of this PR vs the main branch this helps mainting multiple xars and keeping track of changes. Feel free to change it before merging into master.